pkg/gui/views.go

```diff
@@ -94,6 +95,7 @@ func (gui *Gui) orderedViewNameMappings() []viewNameMapping {
 
 		// popups.
 		{viewPtr: &gui.Views.CommitMessage, name: "commitMessage"},
```

Comment:
Let's rename this view (and corresponding label) to `CommitSummary`, which lines up with what github desktop's terminology as well as git kraken's: https://www.gitkraken.com/learn/git/best-practices/git-commit-message#:~:text=Git%20Commit%20Message%20Structure&text=The%20commit%20message%20title%20is,description%20be%20limited%20to%2072.

Reply:
Just to clarify, are you suggesting change just the view or everything else related to commit message? i.e the controller, commit message panel, commit message editor? My fear is the PR could get pretty big and difficult to review :sweat_smile:

Reply:
I am indeed suggesting that :) Happy for that to come in a follow-up PR though